LICENSE-MIT

```diff
@@ -0,0 +1,25 @@
+Copyright (c) 2020 James Waples
+
```

Comment:
Other creates in the Rust ecosystem (including the main rust repo) have removed this line from the MIT license. I think the TLDR was that each author retains the copyright to their contribution and that a global copyright is invalid anyway.

```suggestion
```

Reply:
Makes sense. I've removed it from the license file.